src/StripeTests/Infrastructure/Public/StripeConfigurationTest.cs

```diff
@@ -74,6 +74,32 @@ public void StripeClient_Getter_ThrowsIfClientIsNullAndApiKeyIsEmpty()
             }
         }
 
+        [Fact]
+        public void StripeClient_Getter_ThrowsIfClientIsNullAndClientIdIsEmpty()
```

Comment:
We're not checking that it's throwing, we're just checking if it falls back to `System.Configuration.ConfigurationManager.AppSettings["StripeClientId"]` right? Maybe `StripeClient_Getter_UsesAppSettingsClientIdIfUnset`?